src/deployStack.ts

```diff
@@ -1,15 +1,18 @@
 import createPortainerApi, {StackData} from './api'
 import path from 'path'
 import fs from 'fs'
+import Handlebars from 'handlebars'
 import * as core from '@actions/core'
 
 interface DeployStack {
   portainerHost: string
   username: string
   password: string
   swarmId?: string
+  endpointId: string
```

Comment:
Let's make this one a number instead and then do the `parseInt` and set default to 1 inside `main.ts`.